src/CitMovie.Data/Repositories/BookmarkRepository.cs

```diff
@@ -29,42 +61,6 @@ await _context.Bookmarks
             .Take(pageSize)
             .ToListAsync();
 
-    public async Task<Bookmark> UpdateBookmarkAsync(Bookmark bookmark)
-    {
-        var existingBookmark = await _context.Bookmarks
-            .Where(b => b.BookmarkId == bookmark.BookmarkId)
-            .FirstOrDefaultAsync();
-
-        if (existingBookmark != null && existingBookmark.Note != bookmark.Note)
-        {
-            _context.Entry(existingBookmark).CurrentValues.SetValues(bookmark);
-            await _context.SaveChangesAsync();
-        }
-        return existingBookmark;
-    }
-
-    public async Task<bool> DeleteBookmarkAsync(int bookmarkId)
-    {
-        try
-        {
-            var bookmark = await _context.Bookmarks
-                .Where(b => b.BookmarkId == bookmarkId)
-                .FirstOrDefaultAsync();
-
-            if (bookmark == null) return false;
-
-            _context.Bookmarks.Remove(bookmark);
-            await _context.SaveChangesAsync();
-            return true;
-        }
-        catch
-        {
-            return false;
-        }
-    }
-
-     public async Task<int> GetTotalUserBookmarksCountAsync(int userId)
-    {
-        return await _context.Bookmarks.CountAsync(b => b.UserId == userId);
-    }
-}
```

Comment:
What's the reason for removing this?

Reply:
I did not, I just changed them if you look above.

Reply:
Ahh my bad, though it was part of the move function you added 😅